indra/llcommon/llerror.cpp

```diff
@@ -1623,6 +1628,21 @@ namespace LLError
         }
     }
 
+    void LLUserWarningMsg::showOutOfDisk()
+    {
```

Comment:
If you call this from main thread it will put thread on pause and will prevent viewer from sending 'keep alive' packets. Running out of disk space is a non critical issue, it should not abruptly interrupt user's experience, instead it should be a popup in the corner that recommend to free some space.